e2e/pull/pull.go

```diff
@@ -433,6 +433,47 @@ func orasPushNoCheck(file, ref string) error {
 	return nil
 }
 
+func (c *ctx) testPullDisableCacheCmd(t *testing.T) {
+	cacheDir, err := ioutil.TempDir("", "e2e-imgcache-")
+	if err != nil {
+		t.Fatalf("failed to create temporary directory: %s", err)
+	}
+	defer func() {
+		err := os.RemoveAll(cacheDir)
+		if err != nil {
+			t.Fatalf("failed to delete temporary directory %s: %s", cacheDir, err)
+		}
+	}()
+
+	c.env.TestDir, err = ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("failed to create temporary directory: %s", err)
+	}
+	defer func() {
+		err := os.RemoveAll(c.env.TestDir)
+		if err != nil {
+			t.Fatalf("failed to delete temporary directory %s: %s", c.env.TestDir, err)
+		}
+	}()
+
+	imgName := "testImg.sif"
+	imgPath := filepath.Join(c.env.TestDir, imgName)
+	cmdArgs := []string{"--disable-cache", imgPath, "library://alpine:latest"}
+
+	c.env.RunSingularity(
```

Comment:
Do you need to pass in `cacheDir` so that `singularity` knows what directory to use for the cache? If you do not, will it default to something else?

Reply:
How did I miss that... :(